### PR TITLE
Lax mail dependency version

### PR DESCRIPTION
**Pull request by kruppel**
Allows updates to 2.7+.

### CODE REVIEW

File: savon-multipart.gemspec [hunk 1]
```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.rubyforge_project = s.name
 
   s.add_dependency "savon", "~> 2"
-  s.add_dependency "mail", "~> 2.6"
+  s.add_dependency "mail", "~> 2"
```
Comment (kruppel) [inline savon-multipart.gemspec]: If need be I can change this to, `">= 2.6", "< 3"`. I'm not quite sure what the intent is here though.

**Comment by kruppel** [comment]
@tjarratt Since you'd addressed #20, I only thought it worth asking -- mind taking a look?

File: savon-multipart.gemspec (hunk 1 above)
Comment (jonnyjava) [reply savon-multipart.gemspec]: @kruppel I think your proposal is a good idea, mail 2.5.4 has security issues so is better to allow higher versions.

**Comment by pcai** [comment]
@kruppel thanks!

**Comment by tkusuki** [comment]
@kruppel :tada:

**Comment by henriquegasques** [comment]
Hi @pcai, any plans to release this?

**Comment by FrankSapling** [comment]
@kruppel 
Could you release the changes? The changes can fix the conflict.
Because the minimum require version for mail is out dated. It will come up with a mail version conflict.

**Pull request merged**